### PR TITLE
doc: translate 'explanation-for-github-workflow' into chinese

### PR DESCRIPTION
**Pull request by jieeny**

### CODE REVIEW

File: docs/zh/development/explanation-for-github-workflow.md [hunk 1]
```diff
@@ -1,69 +1,75 @@
...
 ### stale bot 
 ![img_1.png](../../img/development/workflow/img_1.png)
-  
-We use [Close Stale Issues](https://github.com/marketplace/actions/close-stale-issues) .
 
-An issue or PR will be automatically marked as stale if it has not had recent activity in the last 30 days. It will be closed in the next 7 days unless it is tagged (pinned, security, good first issue or help wanted) or other activity occurs.  
+我们将会[关闭超时的Issues](https://github.com/marketplace/actions/close-stale-issues)
 
-**If a community task issue was closed by this bot,then the task can be assigned to others.**  
+如果一个issue或PR在过去30天内没有最近的活动，它将自动标记为陈旧的。它将关闭在未来7天，除非它是被标记的(pinned，security，good first issue或help wanted)或有其他活动发生。
```
Comment (seeflood) [inline docs/zh/development/explanation-for-github-workflow.md:11]: 它将关闭在未来7天  =》 它将在7天后关闭

File: docs/zh/development/explanation-for-github-workflow.md [hunk 2]
```diff
@@ -1,69 +1,75 @@
...
+
+## 2. 持续集成/持续交付
 ![img.png](../../img/development/workflow/img.png)
 ### 2.1. Chore
-#### <1> cla bot
+#### <1> cla bot	
+
+检查贡献者是否签署了贡献许可协议
+
+#### TODO: 在修改proto文件时自动生成新的API引用
```
Comment (seeflood) [inline docs/zh/development/explanation-for-github-workflow.md:24]: API引用  =》 API 文档

File: docs/zh/development/explanation-for-github-workflow.md [hunk 3]
```diff
@@ -1,69 +1,75 @@
...
+#### <2><3> 确保单元测试的覆盖率不会下降
 
-The generated documents are [here](https://github.com/mosn/layotto/blob/main/docs/en/api_reference/runtime_v1.md) and [here](https://github.com/mosn/layotto/blob/main/docs/en/api_reference/appcallback_v1.md)
+具体请查看 https://docs.codecov.com/docs/commit-status#branches
 
-### 2.2. Test
-#### <5> Run unit tests
-#### <5> Check if you have done `go fmt`  
-#### <2><3> Make sure ut coverage ratio won't decrease
+#### TODO: 综合测试
```
Comment (seeflood) [inline docs/zh/development/explanation-for-github-workflow.md:37]: 综合测试 =》 集成测试

File: docs/zh/development/explanation-for-github-workflow.md [hunk 4]
```diff
@@ -1,69 +1,75 @@
...
-### 2.3. Lint
-#### <4> License checker
-We use https://github.com/marketplace/actions/license-eye
+合并于https://github.com/mosn/layotto/pull/247
 
-Merged in https://github.com/mosn/layotto/pull/247
+##### 如何为所有文件自动添加许可证头文件
 
-##### How to add license headers for all files automatically?
 In Layotto directory:
```
Comment (seeflood) [inline docs/zh/development/explanation-for-github-workflow.md:48]: ”In Layotto directory“
这句话可以删了哈

File: docs/zh/development/explanation-for-github-workflow.md [hunk 5]
```diff
@@ -73,9 +79,11 @@ If the title is in either `prefixes` or `regexp`, the check will pass. Otherwise
 }
 ```
 ~~#### TODO: PR body lint?~~
+
 #### TODO: Code style lint
-For example,find out `go xxx()` without `recover`
-We can use go lint and refer to MOSN's configuration  
+举个例子，找出 `go xxx()` 没有 `recover`
+
+我们可以使用 go lint 和 参考MOSN的配置
```
Comment (seeflood) [inline docs/zh/development/explanation-for-github-workflow.md:86]: 我们可以使用 go lint, 参考MOSN的配置

**Review by seeflood — CHANGES_REQUESTED** [review]

**Comment by jieeny** [comment]
Hi, I have a question, ”ubuntu-latest - Go 1.14.13 Expected — Waiting for status to be reported“ , how can I resolve this question?

**Comment by seeflood** [comment]
@jieeny It starts running now.
That's a mechanism of github. A pull request from a first-time contributor can not automatically trigger the CI pipeline untile a maintainer manually starts it.
A little bit annoying :)

File: docs/zh/development/explanation-for-github-workflow.md (hunk 1 above)
Comment (seeflood) [reply docs/zh/development/explanation-for-github-workflow.md:11]: @jieeny hi, it seems that you didn't reslove these review issues?

File: docs/zh/development/explanation-for-github-workflow.md [hunk 6]
```diff
@@ -1,69 +1,75 @@
-This document explain each component of Layotto's github workflow.
+本文档解释了Layotto的github工作流的每个组件。
 
-Note: configuration files of Layotto github workflow are [here](https://github.com/mosn/layotto/tree/main/.github/workflows)
+注意：Layotto的github工作流的配置文档在[这里](https://github.com/mosn/layotto/tree/main/.github/workflows)
```
Comment (seeflood) [inline docs/zh/development/explanation-for-github-workflow.md:3]: 配置文档 => 配置文件

**Review by seeflood — CHANGES_REQUESTED** [review]

**Comment by seeflood** [comment]
Thanks for your help. I modified your branch and fixed the review issues.

**Review by seeflood — APPROVED** [review]
lgtm

**Review by wenxuwan — APPROVED** [review]

**Pull request merged**